<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachReg</class>
 <widget class="QWidget" name="TeachReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>210</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <widget class="QLabel" name="userLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose Username</string>
        </property>
       </widget>
       <widget class="QLabel" name="passLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Create Password</string>
        </property>
       </widget>
       <widget class="QLabel" name="classLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>92</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose Class Code?</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="userEntry">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="passEntry">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="classEntry">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="regButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
